src/main/java/uk/gov/justice/digital/delius/controller/advice/GeneralControllerAdvice.java

```diff
@@ -34,13 +36,15 @@
    }
 
     @ExceptionHandler(WebClientResponseException.class)
+    @Order(2)
```

Comment:
that's quite nice having a default

Reply:
But why isn't the default the subclass?  I don't like it.